src/components/UI/Loader/Loader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Loader as MantineLoader, useMantineTheme } from "@mantine/core";
+import { useStyles, loaderVariants } from "./styles";
+
+interface LoaderProps {
+  size?: "xs" | "sm" | "md" | "lg" | "xl";
```

Comment:
I don't want us to use mantine sizes,refer to typography for sizes and follow the same style